app/models/manual.rb

```diff
@@ -277,7 +277,7 @@ def add_sections_to_manual(manual, edition, published: false)
         Section.find(manual, section_uuid, published: published)
       }
 
-      removed_sections = Array(edition.removed_section_ids).map { |section_uuid|
+      removed_sections = Array(edition.removed_section_uuids).map { |section_uuid|
```

Comment:
Yeah, that was renamed in PR #1052. There are still quite a few variables/methods referring to id instead of uuid when dealing with `Section`s. I plan to fix those separately.